docs/index.md

```diff
@@ -7,32 +7,30 @@ layout: home
     <p>This site will get you up and running with the Python version of the Tableau Document API. The Document API, including the  samples and documentation, are all open source.</p>
     <br />
     <a class="btn btn-primary btn-lg" href="{{ site.baseurl }}/docs" role="button">Get Started</a>&nbsp;&nbsp;
-    <a class="btn btn-primary btn-lg" href="https://github.com/tableau/document-api-python/archive/v0.4.0.zip" role="button">Download</a>
+    <a class="btn btn-primary btn-lg" href="https://github.com/tableau/document-api-python/releases" role="button">Download</a>
 </header>
 
 # Document API Overview
 This repo contains Python source and example files for the Tableau Document API. We're just getting started and have plans to expand what you find here. Help us by submitting feedback, issues, and pull requests!
 
-
 The Document API provides a supported way to programmatically make updates to Tableau workbook and data source files. If you've been making changes to these file types by directly updating the XML--that is, by XML hacking--this SDK is for you.
 
+Features include:
 
->Features include:
->
->- Support for 9.X, and 10.X workbook and data source files
->    - Including TDSX and TWBX files
->- Getting connection information from data sources and workbooks
->    - Server Name
->    - Username
->    - Database Name
->    - Authentication Type
->    - Connection Type
->- Updating connection information in workbooks and data sources
->    - Server Name
->    - Username
->    - Database Name
->- Getting Field information from data sources and workbooks
->    - Get all fields in a data source
->    - Get all fields in use by certain sheets in a workbook
+- Support for 9.X, and 10.X workbook and data source files
```

Comment:
hm, we don't test on any 9.x or 10.x, and it supports beyond that (10.x was 2018, maybe?). Might be better phrased as something like "support for twb, twbx, tde and tdsx files roughly back to 9.x. For hyper files, try the Hyper SDK".

Reply:
@jacalata good point - I was just updating formatting but those references to 9.x/10.x do look weird :)

I'm pushing an update with your suggestion which should modernize the summary a bit.